tests/testthat/test_pipeop_learnercvplus.R

```diff
@@ -0,0 +1,92 @@
+context("PipeOpLearnerCVPlusPlus")
+
+test_that("PipeOpLearnerCVPlus - basic properties", {
+  lrn = mlr_learners$get("regr.featureless")
+  po = PipeOpLearnerCVPlus$new(lrn)
+
+  expect_pipeop(po$clone(), check_ps_default_values = FALSE)
+  expect_data_table(po$input, nrows = 1)
+  expect_data_table(po$output, nrows = 1)
+
+  task = mlr_tasks$get("mtcars")
+  result = train_pipeop(po, list(task = task))
+  expect_null(result[[1L]])
+  expect_list(po$state$cv_models)
+  expect_data_table(po$state$residuals)
+
+  expect_equal(po$predict_type, c("response", "quantiles"))
+  prds = predict_pipeop(po, list(task = task))
+  expect_class(prds, "Prediction")
+  expect_true(all(c("response", "quantiles") %in% names(prds)))
+  expect_true(nrow(prds$quantiles) == nrow(task))
+  expect_equal(dim(predictions$quantiles), c(task$nrow, 3))  # lower and upper quantiles
+
+  expect_error(PipeOpLearnerCVPlus$new())
+
+  expect_pipeop(po("learner_cv_plus", lrn("regr.featureless")))
+})
+
+test_that("PipeOpLearnerCVPlus - param set and values", {
+  skip_if_not_installed("ranger")
+  lrn = mlr_learners$get("regr.ranger")
```

Comment:
do you need 'ranger' for these tests? In that case, you need to load `mlr3learners` (and don't forget to `skip_if_not_installed` it), using `loadNamespace()` or `requireNamespace()`, otherwise the "ranger"-learner is not added to the `lrn()` dictionary.

Reply:
If you can use `rpart` instead of `ranger` then this would not be necessary, because rpart is in the `lrn()` dictionary from the start, without `mlr3learners`.

Reply:
I don't need `ranger`, I updated it to use `rpart` in all tests.